### DISCUSSION TITLE
Heart rate and other BLE and Ant+ device data

### DISCUSSION THREAD

**Discussion by albansuser** [osmandapp/OsmAnd#17380] [General]
I'm interested in knowing if such data embedded in the recorded GPX file can be displayed within OSMAnd yet.

Also, it might be useful if a GPX track file containing the device data could be exported as a TCX file so that it can be used by external applications.

Cheers

**Comment by dmpr0** [comment]
Hi! 
We are working on the task https://github.com/osmandapp/OsmAnd/issues/17200 that will allow to select data type and sensor that will be recorded in GPX. It is available in nightly builds now but not tested yet. OsmAnd doesn't support TCX file format.

**Reply by albansuser** [reply]
Thanks
